ui-src/src/assets/schema.json

```diff
@@ -41,24 +54,54 @@
     },
     "testSuite": {
       "type": "string",
-      "enum": ["Default-1", "Default-2", "Default-3"]
-    },
-    "requestDelay": { "type": "integer", "minimum": 1 },
-    "TPSFreq": { "type": "integer", "minimum": 1 },
-    "rampUsers": { "type": "integer", "minimum": 1 },
-    "rampDelay": { "type": "integer", "minimum": 0 },
-    "testCaseDir": { "type": "string" },
-    "testSuiteDir": { "type": "string" },
-    "baseStatsOutputDir": { "type": "string" },
-    "reportOutputDir": { "type": "string" }
+      "enum": [
+        "Default-1",
```

Comment:
@ConorSmyth do we have a card to get away from populating this field from the schema?